samples/ControlCatalog.Browser/ControlCatalog.Browser.csproj

```diff
@@ -1,13 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="../../src/Browser/Avalonia.Browser/Avalonia.Browser.props" />
-
   <PropertyGroup>
     <TargetFramework>net7.0</TargetFramework>
     <RuntimeIdentifier>browser-wasm</RuntimeIdentifier>
     <WasmMainJSPath>AppBundle/main.js</WasmMainJSPath>
     <OutputType>Exe</OutputType>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <MSBuildEnableWorkloadResolver>true</MSBuildEnableWorkloadResolver>
+    <WasmRuntimeAssetsLocation>./</WasmRuntimeAssetsLocation>
```

Comment:
Can we auto-set it from .props? (if empty)

Reply:
Not really. We don't have a good default value to set.

Reply:
./ - that's a value for .NET 7, also possible with normal SDK in .NET 8
/framework - value for .NET 8, and enforced with StaticWebAssets SDK (no other values are supported)

Reply:
Could we do a check for `UsingMicrosoftNETSdkWebAssembly` to detect if StaticWebAssets is being used?

Reply:
This property isn't always set. I was trying to do so, but in some MSBuild executions it wasn't set. Also note, WASM SDK also runs inner MSBuild process like we do.

Reply:
Moreover, we would still need to modify main.js file depending on this value. 
I.e. first initial `import * from "/_framework/dotnet.js`.  Not sure how we can easily patch user script there.

Reply:
So, imho, it's just easier to keep this change in 11.1 release, and write some migration guide. Especially since we need it anyway, as users need to replace runMainAndExit with runMain on .NET 8 (one of their undocumented breaking changes).